### PR TITLE
Add MaterialEmojiLayoutFactory.

### PR DESCRIPTION
**Pull request by vanniktech**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -112,4 +113,11 @@ wrapper {
...
   task allDependencies(type: DependencyReportTask) {}
-}
+
+  project.plugins.withId("com.android.library") {
+    // TODO replace with https://issuetracker.google.com/issues/72050365 once released.
+    project.android.libraryVariants.all {
+      it.generateBuildConfigProvider.configure { it.enabled = false }
+    }
+  }
+}
```
Comment (mario) [inline build.gradle]: New line.

File: emoji-material/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+POM_NAME=Emoji
+POM_ARTIFACT_ID=emoji-material
+POM_PACKAGING=aar
```
Comment (mario) [inline emoji-material/gradle.properties]: Again :P

**Comment by mario** [comment]
Waaarnings :)

**Comment by mario** [comment]
Maybe you could add the two factories we have to README? Thanks!

**Comment by vanniktech** [comment]
> Maybe you could add the two factories we have to README? Thanks!

Yup will do once I cut the new release. Otherwise people ask why it's missing.

**Comment by vanniktech** [comment]
Waiting for your review @rubengees

**Comment by mario** [comment]
@vanniktech README pls. Even if you just open an issue is fine so we don't forget about it.

**Comment by vanniktech** [comment]
https://github.com/vanniktech/Emoji/issues/407

**Pull request merged**